api/src/main/java/org/eclipse/microprofile/metrics/SimpleTimer.java

```diff
@@ -98,5 +98,33 @@
 
     @Override
     long getCount();
+    
+    /**
+     * Get the maximum recorded time duration of the SimpleTimer for the previously completed minute.
+     * <p>
+     * This represents the highest timed duration in the last complete minute. For example if the last completed
+     * minute recorded durations of 12 seconds, 9 seconds and 1 second the largest value is 12 seconds.
+     * 
+     * If no time duration has been recorded in the last complete minute then 0 is returned.
```

Comment:
I _think_ you might be able to query for values above zero on Prometheus (disclaimer: haven't tried it).
 
Or we can say for an exporter if t he `Duration` is 0 then report at `NaN`. That would provide no data to Prometheus.

Reply:
When there have been no samples in the last minute...
- for SimpleTimer I think 0 could be confusing as a return value (since that's a valid duration).  I would lean towards -1 as a clearer indicator.
- for Prometheus exporter I think our choices would be NaN (as in https://github.com/prometheus/prometheus/issues/860) or just omit the metric from the output. 
- for JSON output ???

Reply:
If no values have been detected then I guess we can report a Duration of -1 nanoseconds.

The JSON exporter can report a null value? NaN doesn't seem to be a valid JSON value.

Reply:
The -1 return values sounds clear.  Will be good to see how tools deal with a NaN in JSON as you suggested offline.

Reply:
null value works as expected in json

Reply:
So if the metric should return -1 nanoseconds, what should be the OpenMetrics output?
I think we don't currently have any tests (or specification) for that. Given that time values are always converted to seconds, this now means that the OpenMetrics export will be like
`application_mysimpletimer_maxTimeDuration_seconds{color="green"} -1.0E-9`
Which is downright ugly.
I think we should go for that NaN

Reply:
@jmartisk , I think we're saying the same thing.  my take is...
java api: -1
OpenMetrics: NaN
json format: ??? (I think @Channyboy is suggesting null?)

Reply:
I'm thinking, is there a reason why not just return null from the Java API?  Null makes it more clear that we don't have data than returning some specific value. 

By the way once we decide how to do this, we probably should do the same for concurrent gauges, they currently export 0 for min/max when there's no data.

Reply:
We could just have the open metrics format be null too. That should work too 😄

Reply:
No, OpenMetrics output can't have null, I tried it and Prometheus complains with

```
level=warn ts=2020-03-26T08:34:45.890177307Z caller=scrape.go:835 component="scrape manager" scrape_pool=local-server-name target=http://localhost:8080/metrics msg="append failed" err="strconv.ParseFloat: parsing \"null\": invalid syntax"
```

Reply:
My mistake, I saw that the Grafana graph showed values when it was suppose to and when it wasn't supposed to. Prometheus indeed throws those warnings.